packages/git/src/node/dugite-git.ts

```diff
@@ -141,14 +141,12 @@ export class CommitDetailsParser extends OutputParser<CommitWithChanges> {
         const chunks = this.split(input, delimiter);
         const changes: CommitWithChanges[] = [];
         for (const chunk of chunks) {
-            const [sha, email, name, timeAsString, authorDateRelative, summary, body, rawChanges] = chunk.trim().split(CommitDetailsParser.ENTRY_DELIMITER);
-            const timestamp = parseInt(timeAsString, 10);
+            const [sha, email, name, timestamp, authorDateRelative, summary, body, rawChanges] = chunk.trim().split(CommitDetailsParser.ENTRY_DELIMITER);
+            // const timestamp = parseInt(timeAsString, 10);
             const fileChanges = this.nameStatusParser.parse(repositoryUri, (rawChanges || '').trim());
             changes.push({
                 sha,
-                author: {
-                    timestamp, email, name
-                },
+                author: { timestamp, email, name },
```

Comment:
It's not required.

Reply:
Never mind. Leave as is but please next time, do not change if not necessary. It does not add much value to the code and it is easier to review.